pkg/distsql/select_result.go

```diff
@@ -603,6 +603,13 @@ func (r *selectResult) Close() error {
 	if respSize > 0 {
 		r.memConsume(-respSize)
 	}
+	if unconsumed, ok := r.resp.(copr.HasUnconsumedCopRuntimeStats); ok && unconsumed != nil {
+		unconsumedCopStats := unconsumed.CollectUnconsumedCopRuntimeStats()
```

Comment:
Since here handle unconsumed cop stats, could we directly release all the stats instead of release every one stats by loop?